ui/tests/acceptance/secrets/backend/azure/azure-configuration-test.js

```diff
@@ -112,7 +112,8 @@ module('Acceptance | Azure | configuration', function (hooks) {
     });
 
     module('create', function () {
-      test('it should save azure account accessType options', async function (assert) {
+      test('it should save azure account options', async function (assert) {
+        // there are no azure specific options that can be returned from the API so confirm the generic options are saved.
```

Comment:
Very similar acceptance tests that we do for GCP as it too cannot check if GCP specific options where set.